darts/timeseries.py

```diff
@@ -345,6 +345,45 @@ def from_times_and_values(times: Optional[Union[pd.DatetimeIndex, pd.RangeIndex]
                                          fill_missing_dates=fill_missing_dates,
                                          freq=freq)
 
+    @staticmethod
+    def from_times_and_values_probabilistic(times: Optional[Union[pd.DatetimeIndex, pd.RangeIndex]],
```

Comment:
Actually could we re-use `from_times_and_values()` and have it behave differently depending whether the dimensionality of `values` is 3 (like here), 2 (like the current `from_times_and_values() - which we have no reason to keep dependent on DataFrame), or 1 (in which case we assume it's a deterministic series with 1 component).

Reply:
Good point, done! Doing this I was also able to simplify the `TimeSeries.from_values()` method. Let me know if the changes make sense